diff -ur eog-43.1/data/org.gnome.eog.desktop.in.in eog-43.1.new/data/org.gnome.eog.desktop.in.in
--- eog-43.1/data/org.gnome.eog.desktop.in.in	2022-10-22 11:49:04.000000000 -0700
+++ eog-43.1.new/data/org.gnome.eog.desktop.in.in	2022-11-16 15:00:32.553832149 -0800
@@ -11,7 +11,7 @@
 Type=Application
 Categories=GNOME;GTK;Graphics;2DGraphics;RasterGraphics;Viewer;
 X-GNOME-DocPath=eog/eog.xml
-MimeType=image/bmp;image/gif;image/jpeg;image/jpg;image/pjpeg;image/png;image/tiff;image/webp;image/x-bmp;image/x-gray;image/x-icb;image/x-ico;image/x-png;image/x-portable-anymap;image/x-portable-bitmap;image/x-portable-graymap;image/x-portable-pixmap;image/x-xbitmap;image/x-xpixmap;image/x-pcx;image/svg+xml;image/svg+xml-compressed;image/vnd.wap.wbmp;image/x-icns;
+MimeType=image/bmp;image/gif;image/jpeg;image/jpg;image/pjpeg;image/png;image/tiff;image/webp;image/x-bmp;image/x-gray;image/x-icb;image/x-ico;image/x-png;image/x-portable-anymap;image/x-portable-bitmap;image/x-portable-graymap;image/x-portable-pixmap;image/x-xbitmap;image/x-xpixmap;image/x-pcx;image/svg+xml;image/svg+xml-compressed;image/vnd.wap.wbmp;image/x-icns;image/jxl
 # Extra keywords that can be used to search for eog in GNOME Shell and Unity
 # TRANSLATORS: Search terms to find this application.
 #              Do NOT translate or localize the semicolons!
